api/api_v01.rb

```diff
@@ -43,6 +43,13 @@ class ApiV01 < Grape::API
           in: 'query'
         }
       },
+      security: [{
+        api_key_query_param: [],
+      }],
+      schemes: [
+        'http',
+        'https'
+      ],
```

Comment:
This commit `Move security definition` need to be merged to the main grape commit (since without this commit grape-swagger doesn't work as expected).